platform/darwin/src/MGLMapSnapshotter.mm

```diff
@@ -162,14 +179,18 @@ - (void)startWithQueue:(dispatch_queue_t)queue completionHandler:(MGLMapSnapshot
 
         strongSelf.loading = false;
 
+        if (!strongSelf.completion)
```

Comment:
FWIW, mbgl-core prefers the verbose guard style:

```
if (condition) {
   return;
}
```

I think the reasoning is basically to standardize control flow syntax as much as possible so that people skimming code don't miss a hidden statement tacked onto the end of a line. Also it's handy to be able to quickly put a breakpoint that hits on the return condition.

But I don't have strong evangelical feelings here and it's reasonable for our Objective-C code to follow its own style.

Reply:
I tend to prefer multi-line for debugging purposes. Slip of the finger, not adding the braces.

Reply:
We have been following the one line syntax or using brackets for one line ifs. While this seem minor it can hide potential bugs in the future. I think either
```
if ( condition ) {
     return;
}
```
or
`if ( condition ) returns;`
Achieves the goal.